### ISSUE TITLE
Intro to Deep Learning

### ISSUE THREAD

**Issue by drakearch** [drakessn/Kaggle-Courses#26]
Use TensorFlow and Keras to build and train neural networks for structured data.
- [x] A Single Neuron
- [x] Deep Neural Networks
- [x] Stochastic Gradient Descent
- [x] Overfitting and Underfitting
- [x] Dropout and Batch Normalization
- [x] Binary Classification
- [x] Bonus 1: Detecting the Higgs Boson With TPUs
